package.json

```
@@ -12,7 +12,10 @@
   },
   "devDependencies": {
     "@stencil/core": "^2.0.0",
-    "@stencil/router": "^1.0.1"
+    "@stencil/router": "^1.0.1",
+    "@types/jest": "^26.0.24",
+    "jest": "^27.0.6",
```

Stencil doesn't support Jest 27 at this moment. It's something the team is working on, but with the latest version of Stencil that's publicly available, running `npm test` will fail:
```
[ ERROR ]  runJest: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './build/cli/args' is not defined by
           "exports" in /Users/ryan/workspaces/stencil-app-starter/node_modules/jest-cli/package.json
```
Would you mind downgrading this to `jest@26.6.3`:
```suggestion
    "jest": "^26.6.3",
```